### PR TITLE
Issue #1028: Make the layout list more useful and informative

### PR DESCRIPTION
**Pull request by docwilmot**
Fixes https://github.com/backdrop/backdrop-issues/issues/1028#issuecomment-114559464

### CODE REVIEW

File: core/modules/layout/js/layout.admin.js [hunk 1]
```diff
@@ -58,6 +58,15 @@ Backdrop.behaviors.layoutConfigure = {
...
   attach: function(context) {
+    $('.layout-conditions').find('.layout-conditions-list').hide();
+    // Toggle the conditions info on layout list page.
+    $('.conditions-toggle').click(function(e) {
+      var $conditions = $(this).parent().find('.layout-conditions-list').toggle();
+      $(this).toggleClass('more');
+      e.preventDefault();
+      e.stopPropagation();
+    });
+
```
Comment (klonos) [inline core/modules/layout/js/layout.admin.js:69]: trailing space there.

File: core/modules/layout/layout.theme.inc [hunk 2]
```diff
@@ -46,8 +46,37 @@ function theme_layout_reorder_layouts($variables) {
  */
 function theme_layout_info($variables) {
   $layout = $variables['layout'];
+  $info = layout_get_layout_info($layout->layout);
+
```
Comment (klonos) [inline core/modules/layout/layout.theme.inc:50]: ...and here.

File: core/modules/layout/layout.theme.inc [hunk 3]
```diff
@@ -46,8 +46,37 @@ function theme_layout_reorder_layouts($variables) {
  */
 function theme_layout_info($variables) {
   $layout = $variables['layout'];
+  $info = layout_get_layout_info($layout->layout);
+  
+  // Buuild template thumbnail preview
+  $preview_path = $info['path'] . '/' . $info['preview'];
+  $preview = theme('image', array('uri' => $preview_path));
+
```
Comment (klonos) [inline core/modules/layout/layout.theme.inc:54]: ...and here.

File: core/modules/layout/layout.theme.inc [hunk 4]
```diff
@@ -46,8 +46,37 @@ function theme_layout_reorder_layouts($variables) {
...
+  $preview_path = $info['path'] . '/' . $info['preview'];
+  $preview = theme('image', array('uri' => $preview_path));
+  
+  // Build a list of visibility conditions on this layout
+  $conditions = array();
+  foreach ($layout->conditions as $access_key => $layout_access) {
+    $conditions[] = $layout_access->summary();
+  }
+  $conditions_list = theme('item_list', array('items' => $conditions));
+
```
Comment (klonos) [inline core/modules/layout/layout.theme.inc:61]: ...and here.

File: core/modules/layout/layout.theme.inc [hunk 5]
```diff
@@ -46,8 +46,37 @@ function theme_layout_reorder_layouts($variables) {
...
+  // Build a list of visibility conditions on this layout
+  $conditions = array();
+  foreach ($layout->conditions as $access_key => $layout_access) {
+    $conditions[] = $layout_access->summary();
+  }
+  $conditions_list = theme('item_list', array('items' => $conditions));
+  
+  // Create a link to the settings page
+  $settings_link = l($info['title'], 'admin/structure/layouts/manage/' . $layout->name . '/settings');
+
```
Comment (klonos) [inline core/modules/layout/layout.theme.inc:64]: ...and here.

File: core/modules/layout/layout.theme.inc (hunk 3 above)
Comment (docwilmot) [inline core/modules/layout/layout.theme.inc:54]: You missed "Buuild".

**Comment by klonos** [comment]
> ...You missed "Buuild".

Just saw that. Ha :smile:

**Comment by quicksketch** [comment]
Replaced by https://github.com/backdrop/backdrop/pull/1385.

**Pull request closed without merging**